grails-app/controllers/appforliteracy/sampleModuleController.groovy

```diff
@@ -0,0 +1,14 @@
+package appforliteracy
+
+class sampleModuleController {
+
+    def index() { }
+
+    def sampleModuleInputDisplay() {
+        print "Here"
+    }
+
+    def loadFile () {
+
+    }
+}
```

Comment:
Will be filled in with a sample module that displays the input list and returns an output of what the longest word in that list is. This is intended to be a stub that will be replaced by a much richer sample module later.